src/core.ts

```diff
@@ -114,41 +150,40 @@ export function toTokens(storedTokens: StoredTokens): Tokens {
   return { ...tokens, expiresIn };
 }
 
-/**
- * Retrieves the token for the given session ID.
- * Before retrieval, the stored token is converted to a normal token using {@linkcode toTokens}.
- */
-export async function getTokensBySession(
+const TOKENS_PREFIX = "tokens";
```

Comment:
How about making this configurable by users?

Reply:
I don't think it should be.